src/IB/IBRedundantInitializer.cpp

```diff
@@ -1496,8 +1496,11 @@ IBRedundantInitializer::getVertexBdryMassSpec(const std::pair<int, int>& point_i
 const std::vector<double>&
 IBRedundantInitializer::getVertexDirectors(const std::pair<int, int>& point_index, const int level_number) const
 {
-    if (!d_init_director_and_rod_on_level_fcn)
+    if (static_cast<size_t>(point_index.second) >= d_directors[level_number][point_index.first].size())
     {
+        TBOX_WARNING("Could not find enough director vectors on level " << level_number << " for structure "
+                                                                        << point_index.first << "\n"
+                                                                        << "Using the standard basis.\n");
```

Comment:
This seems a little opaque; maybe something like:
```suggestion
                                                                        << "Initializing director vectors to the standard basis.\n");
```
Also, is it all directors for the structure?
```suggestion
                                                                        << "Initializing all director vectors for structure " << point_index.first  << " on level " << level_number << " to the standard basis.\n");
```